docs/kb/semgrep-ci/scan-monorepo-in-parts.md

```diff
@@ -17,7 +17,7 @@ As such, it can be helpful to scan a monorepo in parts for multiple reasons:
 
 ## How to configure Semgrep in CI to split up a monorepo
 
-When scanning a repo with Semgrep in CI, the base command is `semgrep ci`. To understand this default setup for your source code manager (SCM) and CI provider, see [Getting started with Semgrep in continuous integration (CI)](/docs/semgrep-ci/overview/).
+When scanning a repo with Semgrep in CI, the base command is `semgrep ci`. To understand this default setup for your source code manager (SCM) and CI provider, see [Getting started with Semgrep in continuous integration (CI)](/docs/semgrep-ci/overview).
```

Comment:
```suggestion
When scanning a repo with Semgrep in CI, the base command is `semgrep ci`.
```
Suggest removing that entire line, because that doc now redirects to a pure procedural doc without any "how does semgrep ci work" type of information. This KB article itself does a good job giving steps to split up the monorepo.